hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/models/LockProviderHeartbeatManager.java

```diff
@@ -228,7 +228,7 @@ private void heartbeatTaskRunner(Thread threadToMonitor) {
   private boolean executeHeartbeat(Thread threadToMonitor) {
     // Check if monitored thread is dead
     if (!threadToMonitor.isAlive()) {
-      logger.warn("Owner {}: Monitored thread is no longer alive.", ownerId);
+      logger.info("Owner {}: Monitored thread is no longer alive.", ownerId);
```

Comment:
this i believe should be a warn situation. definitely a rare case, but something we want to notice in the logs vs info or error. we exit gracefully, but it's indicative of a larger problem if the writer thread exits without ending the transaction.